### PR TITLE
17067: IVL Notice Template Cover Page 

### PR DESCRIPTION
**Pull request by Battula**
|Field|Value|
|-----|-----|
| Redmine Ticket Number | [17067](https://devops.dchbx.org/redmine/issues/17067) |
| App-Dev Road Map # | n/a |
| Start Date | n/a |
| Due Date | n/a  |
| App-Dev Priority | UP-Disqueued |
| Development Story Points | n/a |
| Peer Review Story Points | n/a |
| Ticket Author | @Ckellem |

### CODE REVIEW

File: app/views/layouts/ivl_pdf_notice.html.erb [hunk 1]
```diff
@@ -0,0 +1,200 @@
...
+      /* RESET */
+      #outlook a {
+        padding:0
+      }
+
+      @font-face {
+        font-family: "Open Sans";
+        font-style: normal;
+        font-weight: normal;
+        src: url("<%= asset_data_base64('open-sans-hebrew-light/OpenSansHebrew-Light.ttf') %>") format('opentype');
```
Comment (davidplappert) [inline app/views/layouts/ivl_pdf_notice.html.erb]: Remove every line but the font-family in this section, and run your test again.

File: app/views/notices/shared/footer_ivl.html.erb [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+  <meta content="IE=edge" http-equiv="X-UA-Compatible" />
+  <meta content="telephone=no" name="format-detection" />
+
+  <style type="text/css">
+
+    @font-face {
+      font-family: 'Open Sans';
+      font-style: normal;
+      font-weight: 300;
+      src: url("<%= Rails.root.join('app', 'assets', 'fonts/Open_Sans', 'OpenSans-Regular.ttf') %>") format('opentype')
```
Comment (davidplappert) [inline app/views/notices/shared/footer_ivl.html.erb]: same thing here: remove everything but font-family.

Comment (Battula) [inline app/views/notices/shared/footer_ivl.html.erb]: This didn't solve the issue.

File: app/views/layouts/ivl_pdf_notice.html.erb (hunk 1 above)
Comment (Battula) [inline app/views/layouts/ivl_pdf_notice.html.erb]: This change didn't solve the issue.

**Comment by Kevin-Wei** [comment]
addressed via https://github.com/dchbx/enroll/pull/1502

**Pull request closed without merging**